---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "Integrative Genetic and Genomic Analyses in the Inflammatory Bowel Disease"
summary: ""
authors: ["Shehzad Sheikh"]
tags: [R01, Ancillary, NIDDK]
categories: []
date: 2021-04-12T11:14:32-05:00

# Optional external URL for project (replaces project detail page).
external_link: ""

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: true

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""
---

The inflammatory bowel diseases (IBD), composed of Crohn's disease (CD) and ulcerative colitis (UC), result from an inappropriately directed inflammatory response to the enteric microbiota in a genetically susceptible host. Genome wide association studies (GWAS) have linked 163 specific single nucleotide polymorphisms (SNPs) to IBD disease pathogenesis. Within these associated loci, there are over 5000 additional SNPs that are in linkage disequilibrium (LD) with the identified SNPs, and it is not known which of these contribute to IBD. Most of these SNPs map to non-coding regions of the genome, suggesting that many variants contribute to IBD by modifying gene regulatory element activity. We hypothesize that DNA variation that impacts gene regulatory element activity significantly contributes to the differential host response to microbial stimuli between normal and IBD individuals. The primary goal of this proposal is to identify genetic, regulatory activity, and gene expression changes that significantly contribute to IBD at molecular and clinical levels. Genomic regions of nucleosome-depleted, open chromatin have been linked to all types of regulatory elements, and genome-wide assays to detect these are now available. In this proposal we will determine open chromatin status, transcription levels, and genotypes for a panel of comprehensively phenotyped CD patient tissue samples obtained from the non-inflamed section of the ascending colon. Similar data will be generated for non-IBD controls for comparison. Using these data, we will determine regulatory elements with genetically driven differential activity across a host of clinical and molecular IBD phenotypes.
