---
# Adapted from themes/academic/archetypes/project/index.md
title: "Cedars-Sinai"
subtitle: "IBD Genetics Research Center"
pi: "Dermot McGovern, MD, PhD, FRCP(Lon), FACG"
pi_titles: ["Principal Investigator", "Professor, Medicine and Biomedical Sciences", "Director, Translational Research in the Inflammatory Bowel and Immunobiology Research Institute", "Director, Precision Health, Cedars-Sinai"]
picture: "/img/cedars/dermot-mcgovern.jpeg"
#bio: ["The McGovern Laboratory studies the effects of genetic variation in the inflammatory bowel diseases (IBD). Also called the Translational Genomics Group (TGG), the McGovern Lab is focused on identifying genetic variants associated with both Crohn&#39;s disease and ulcerative colitis and then investigating their functional effects and also their role in determining disease behavior and natural history, including response to therapy. In addition to gene discovery in Northern European ancestry populations, the McGovern Lab focuses on gene discovery in African American, Hispanic, Ashkenazi Jewish, and East Asian populations. These studies are designed, in part, to discover new therapies for IBD as well as to enhance our ability to apply a personalized approach to managing our patients in the clinic and to ensure that all populations benefit from these advances. The McGovern Lab also investigates the role of the microbiome in IBD and particularly the relationship between genetic variation in the host (human) and microbiome-related changes. The TGG is an integral part of the National Institute of Diabetes and Digestive and Kidney Diseases and International IBD Genetics Consortia as well as a member of the Very Early Onset IBD Consortium. Work in the McGovern Laboratory includes significant statistical and computational biology components, and the lab is developing novel methods for integrating multi-omic datasets together with clinical metadata."]
tags: []
categories: []
date: 2022-10-24

# Optional external URL for PI
external_link: "https://www.cedars-sinai.org/research/labs/mccgovern.html"

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.

publications:
    count: 12
    order: "desc"
---

<!-- manual color setting needed due to wowchemy page-wrapper breaking change -->
<body style="background-color: rgb(54, 70, 90);">

# Lead Clinicians

- Gil Melmed, MD
- Phil Fleshner, MD (surgeon)
- Shervin Rabizadeh, MD and David Ziring, MD (Pediatrics)
- Cindy Kallman, MD (Radiology)
- Elena Chang, MD (Pathology)
- Eric Vasiliauskas, MD
- Nirupama Bonthala, MD
- Phillip Gu, MD
- Andres Yarur, MD
- University of Southern California: Sarah Sheibanu, MD 
- University of Southern California: Bing Zhang, MD, MAS
- Children’s Hospital Los Angeles: Sonia Michail, MD, FAAP, AGAF
- UC San Diego: Siddharth Singh, MD, MS
- Houston Methodist: Bincy P. Abraham, MD, MS, AGAF, FACG
- The University of Mississippi Medical Center: Sarah C. Glover, DO AGAF
- University of Nevada, Reno: Mark S. Riddle, MD, DrPH, FISTM

# Lab and Project Management

- Emebet Mengesha (Lab Manager)
- Shell Shaohong, MD (Phenotyper)

# Clinical Coordinators

- Mary Hanna
- Abdul Khan
- Anthony Navarro

# Analysts

- Dalin Li, PhD
- Talin Haritunians, PhD
- Shishir Dube, PhD
- Sarfaraz Lalani
- Minsuk Kim, PhD
- Dylan Shiramizu
- Ruowang Li, PhD

---

### NIDDK IBDGC-related goals

#### Specific Aims

**Aim 1:** Characterize genetic variation and host-microbiome interactions associated with IBD in Hispanics. A) We will build the largest cohort of Hispanic IBD subjects. (Collaboration: MiaLAtinX Consortium) B) We will define common and rare variation associated with IBD in Hispanics including variants identified through admixture mapping (collaboration: Mark Daly). C) Using biomarker-based inference to quantitate peripheral immune responses to up to 1,800 common human gut-resident bacteria and IBD-associated serologies, we will identify host-microbiome in Hispanic IBD.

**Aim 2:** Identify molecular causes of anti-TNF responsiveness through an integrated resource of both genetic and multi-modal single-cell data. In a prospective cohort of CD patients, we will determine the single-cell proteogenomic signatures and longitudinal single-cell multi-modal transcriptional responses associated with outcomes from anti-TNF therapy.

**Aim 3:** Use of human intestinal epithelial cell culture systems to screen for function of new IBD susceptibility loci. A substantial number of genes with coding variants associated with CD are highly expressed in the intestinal epithelium. We will generate human intestinal organoids and use novel assays to determine functional consequences of coding-changing variants associated with IBD.

#### Ancillary liaisons

Jon Braun; Cedars-Sinai Medical Center: Biomarking IBD patient-specific disease features using the epithelial antigenic peptidome (R21 DK127189-01)

---

#### Additional resources at Cedars-Sinai

- MIRIAD Biorepository: Contains over 15K IBD research subjects (includes extended families) for whom ~12,000 have whole exome or whole genome sequence data and GSA data. All subjects are consented for ‘call-backs’ for additional sampling as well as access to medical charts, radiology, histopathology etc.
- EBV Transformed Lymphoblastoid Cell Line Bank in ~15,000 subjects with corresponding genetic data available in ~12,000 subjects and including capabilities to ‘drive’ cell lines to IPSCs as well as ‘Gut-on-a-Chip’ technology.
- Computational Pathology and Advanced Approaches to Imaging: Drs. Arkadiusz Gertych and Shishir Dube are bringing advanced computational approaches to histopathology and radiology, respectively, and leading efforts to integrate this with other ‘-omic’ data and clinical data across the GRCs.
- McGovern Lab Genotyping Facility: The lab is fully equipped with Illumina technology and performs genotyping and thorough quality control on variety of arrays depending on project needs.
- Applied Genomics, Computational & Translational (AGCT) Core:  Novaseq 6000, NextSeq 500, and MiSeq suite of Illumina next-generation sequencers provides researchers extreme flexibility in a cost-efficient manner. They offer a wide range of read capacity, output, run times, and costs for different sequencing applications.  In addition, the core has single cell systems including 10x GENOMICS CHROMIUM CONTROLLER, 10X GENOMICS CHROMIUM CONNECT, BD BIOSCIENCE RHAPSODY SINGLE CELL SYSTEMS, MISSION BIO TAPESTRI, and several tissue and single cell sample preparation platforms.

#### Major interests

- Genetic and genomic studies to identify risk factors for IBD and also to study pleiotropic effects between IBD and other traits immune-mediated, metabolic, infectious, and neurological traits. (Additional collaborators: SARS-CoV2 (CORALE and CLARITY consortia); Parkinson’s Disease (Clive Svendsen))
- Genotype x Phenotype studies, including natural history, disease complications, and response to therapies (Collaborators: VEOIBD & SHARE Consortia)
- Genotype x Microbiome Studies (Additional Collaborators: Suzanne Devkota and David Underhill)
- Non-Northern European population studies: Ashkenazi Jews, East Asians, African Americans, Hispanics. (Additional Collaborators: Asan Medical Center, Korea; Tohoku University, Japan; Hailiang Hang; Children’s Hospital of LA; MiLAtinX Consortium, Marie Abreu; Esther Torres, University of Puerto Rico)
- Pharmacogenomics (Additional Collaborators: PANTS Consortium, Tariq Ahmad, Mark Daly)
- Sex-Specific Influences on IBD pathophysiology and Natural History (IIBDGC)

