---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: Common and rare variant prediction and penetrance of IBD in a large, multi-ethnic,
  health system-based biobank cohort
subtitle: ''
summary: ''
authors:
- Kyle Gettler
- Rachel Levantovsky
- Arden Moscati
- Mamta Giri
- Yiming Wu
- Nai-Yun Hsu
- Ling-Shiang Chuang
- Aleksejs Sazonovs
- Suresh Venkateswaran
- Ujunwa Korie
- Colleen Chasteau
- NIDDK IBDGC UK IBD Genetics Consortium
- Richard H. Duerr
- Mark S. Silverberg
- Scott B. Snapper
- Mark J. Daly
- Dermot P. McGovern
- Steven R. Brant
- Subra Kugathasan
- Carl A. Anderson
- Yuval Itan
- Judy H. Cho
tags:
- '"IBD"'
- '"PRS"'
- '"VEO-IBD"'
categories: []
date: '2020-12-01'
lastmod: 2021-02-15T06:36:05-06:00
featured: true
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-02-15T12:57:09.976242Z'
publication_types:
- '2'
abstract: 'BACKGROUND AND AIMS: Polygenic risk scores (PRS) may soon be used to predict
  inflammatory bowel disease (IBD) risk in prevention efforts. We leveraged exome-sequence
  and SNP array data from 29,358 individuals in the multi-ethnic, randomly-ascertained
  health system-based BioMe biobank to define effects of common and rare IBD variants
  on disease prediction and pathophysiology. METHODS: PRS were calculated from European,
  African-American, and Ashkenazi Jewish (AJ) reference case-control studies, and
  a meta-GWAS run using all three association datasets. PRS were then combined using
  regression to assess which combination of scores best predicted IBD status in European,
  AJ, Hispanic, and African American cohorts in BioMe. Additionally, rare variants
  were assessed in genes associated with very early onset IBD (VEO-IBD), by estimating
  genetic penetrance in each BioMe population. RESULTS: Combining risk scores based
  on association data from distinct ancestral populations improved IBD prediction
  for every population in BioMe and significantly improved prediction among European
  ancestry UK Biobank individuals. Lower predictive power for non-Europeans was observed,
  reflecting in part substantially lower African IBD case-control reference sizes.
  We replicated associations for two VEO-IBD genes, ADAM17 and LRBA, with high dominant
  model penetrance in BioMe. Autosomal recessive LRBA risk alleles are associated
  with severe, early-onset autoimmunity; we show that heterozygous carriage of an
  African-predominant LRBA protein-altering allele is associated with significantly
  decreased LRBA and CTLA-4 expression with T cell activation. CONCLUSIONS: Greater
  genetic diversity in African populations improves prediction across populations,
  and generalizes some VEO-IBD genes. Increasing African-American IBD case-collections
  should be prioritized to reduce health disparities and enhance pathophysiologic
  insight.'
publication: '*Gastroenterology*'
doi: 10.1053/j.gastro.2020.12.034
---

{{< figure src="/img/publication/gettler-common-2020/fig3.png" title="Inclusion of association summary information from European, African American, and Ashkenazi Jewish datasets improve prediction of IBD status relative to prediction using only the most relevant association data for each population.  This holds true for IBD prediction for individuals of European ancestry in both the BioMe and UK Biobank cohorts, Hispanic ancestry in BioMe, and Ashkenazi Jewish ancestry in BioMe.  Prediction of African American IBD is challenging both because association studies among African American individuals have been limited by low sample size, and because of greater genetic diversity." >}}
{{< figure src="/img/publication/gettler-common-2020/fig5.png" title="Here we validated that a very rare variant identified to have high genetic penetrance when considering UC risk also had an impact on expression of LRBA and CTLA4 in heterozygous carriers.  We also show that treatment with chloroquine was able to partially rescue CTLA4 expression in these carriers, suggesting that it may be useful for personalized treatment of UC." >}}
