---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "Crohnâ€™s Disease: Influence of Host Genetics on the Composition and Function of the Gut Microbial Communities"
summary: ""
authors: ["Ramnik Xavier", "Liason: Dermot McGovern"]
tags: [R01, Ancillary, NIDDK]
categories: []
date: 2021-04-12T11:15:57-05:00

# Optional external URL for project (replaces project detail page).
external_link: ""

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""
---
The interplay between a host's genetics, immune system, and intestinal microbial communities is a dynamic force influencing health and disease status. This proposal will assess how genetic variants associated with IBD shape the microbiota and the functional consequences that alter host responses and culminate pathologic inflammation. We focus on autophagy-linked genetic variants to determine whether genetic subsets of Crohn's disease (CD) are associated with specific changes in the luminal microbiome. We will explore microbial community structure, function and metabolism, and the microbiome's interaction with its host in CD. We will leverage the extensive resources and detailed phenotypic information of the IBDGC as well as several other IBD registries (Massachusetts General Hospital, Boston Children's Hospital, and the Ocean State Crohn's and Colitis Area Registry, an inception cohort in Rhode Island) to enroll subjects with ATG16L1, IRGM, and NOD2 disease-associated variants, as well as healthy individuals with and without these risk alleles. Using 16S ribosomal sequencing of luminal samples, we will apply novel biostatistical techniques to associate organisms or clades within the microbiome with CD status and host genetics. Subsequent whole genome sequencing and transcriptomics will allow us to advance beyond categorization of disease-associated bacteria to understand the functional implications of the dysbiosis associated with CD. Utilizing mouse models of the autophagy allele variants, we will confirm the genetic impact of the microbiome by performing association in gnotobiotic mice with the human study samples and investigate the host response to these microbial changes. CD is fundamentally shaped by intestinal microbial factors, which may provoke the disease in individuals with defined genetic risks. The specific roles of organisms or metabolism in the intestinal microbiome during CD remain poorly defined and are emerging frontiers in deciphering the pathophysiology of the disease. The results anticipated from these studies will advance our understanding of CD by determining how the interactions and influences of host genetics, the microbiome and the immune response contribute to Crohn's disease. PUBLIC HEALTH RELEVANCE: This proposal aims to identify microbial factors that contribute to IBD in patients with genetic polymorphisms in autophagy genes. The project is consistent with the goals of the Human Microbiome Project and has the potential to be truly transformative by contributing to a fundamental knowledge gap regarding the cause of IBD. The results could transform our understanding of relationships between microbes and humans and can lead to innovative diagnostic tests and future treatments.
