---
# Documentation: https://wowchemy.com/docs/managing-content/

title: A Disease-Associated Microbial and Metabolomics State in Relatives of Pediatric
  Inflammatory Bowel Disease Patients
subtitle: ''
summary: ''
authors:
- Jonathan P. Jacobs
- Maryam Goudarzi
- Namita Singh
- Maomeng Tong
- Ian H. McHardy
- Paul Ruegger
- Miro Asadourian
- Bo-Hyun Moon
- Allyson Ayson
- James Borneman
- Dermot P. B. McGovern
- Albert J. Fornace
- Jonathan Braun
- Marla Dubinsky
tags:
- '"Inflammatory Bowel Disease"'
- '"AUC"'
- '"area under the curve"'
- '"CD"'
- '"Crohnâ€™s disease"'
- '"Family Cohort"'
- '"IBD"'
- '"inflammatory bowel disease"'
- '"LC/MS"'
- '"liquid chromatography/mass spectrometry"'
- '"Metabolomics"'
- '"Microbiome"'
- '"OTU"'
- '"operational taxonomic unit"'
- '"PCoA"'
- '"principal coordinates analysis"'
- '"PCR"'
- '"polymerase chain reaction"'
- '"rRNA"'
- '"ribosomal RNA"'
- '"ToFMS"'
- '"time-of-flight mass spectrometry"'
- '"UC"'
- '"ulcerative colitis"'
- '"UPLC"'
- '"ultra-performance liquid chromatography"'
categories: []
date: '2016-11-01'
lastmod: 2021-11-01T08:41:25-04:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-11-01T12:41:25.112180Z'
publication_types:
- '2'
abstract: "BACKGROUND & AIMS: Microbes may increase susceptibility to inflammatory\
  \ bowel disease (IBD) by producing bioactive metabolites that affect immune activity\
  \ and epithelial function. We undertook a family based study to identify microbial\
  \ and metabolic features of IBD that may represent a predisease risk state when\
  \ found in healthy first-degree relatives. METHODS: Twenty-one families with pediatric\
  \ IBD were recruited, comprising 26 Crohn's disease patients in clinical remission,\
  \ 10 ulcerative colitis patients in clinical remission, and 54 healthy siblings/parents.\
  \ Fecal samples were collected for 16S ribosomal RNA gene sequencing, untargeted\
  \ liquid chromatography-mass spectrometry metabolomics, and calprotectin measurement.\
  \ Individuals were grouped into microbial and metabolomics states using Dirichlet\
  \ multinomial models. Multivariate models were used to identify microbes and metabolites\
  \ associated with these states. RESULTS: Individuals were classified into 2 microbial\
  \ community types. One was associated with IBD but irrespective of disease status,\
  \ had lower microbial diversity, and characteristic shifts in microbial composition\
  \ including increased Enterobacteriaceae, consistent with dysbiosis. This microbial\
  \ community type was associated similarly with IBD and reduced microbial diversity\
  \ in an independent pediatric cohort. Individuals also clustered bioinformatically\
  \ into 2 subsets with shared fecal metabolomics signatures. One metabotype was associated\
  \ with IBD and was characterized by increased bile acids, taurine, and tryptophan.\
  \ The IBD-associated microbial and metabolomics states were highly correlated, suggesting\
  \ that they represented an integrated ecosystem. Healthy relatives with the IBD-associated\
  \ microbial community type had an increased incidence of elevated fecal calprotectin.\
  \ CONCLUSIONS: Healthy first-degree relatives can have dysbiosis associated with\
  \ an altered intestinal metabolome that may signify a predisease microbial susceptibility\
  \ state or subclinical inflammation. Longitudinal prospective studies are required\
  \ to determine whether these individuals have a clinically significant increased\
  \ risk for developing IBD."
publication: '*Cellular and Molecular Gastroenterology and Hepatology*'
doi: 10.1016/j.jcmgh.2016.06.004
---
