---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: "Ancillary Studies"
subtitle: ""
summary: ""
authors: []
tags: []
categories: []
date: 2020-08-31T06:47:40-05:00
lastmod: 2020-08-31T06:47:40-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
---

{{% toc %}}

## Current R01 Projects

### New computational, transcriptional, and genome editing approaches to the biology of inflammatory bowel disease

- Greg Gibson, Georgia Institute of Technology
- Liaison: Judy Cho

Over the past decade, the NIDDK IBDGC (Inflammatory Bowel Disease Genetics
Consortium) has generated extraordinary datasets in support of genetic
analysis of the onset, progression, and therapeutic response to Crohn's
disease and ulcerative colitis. This Ancillary project will complement ongoing
IBDGC research by providing parallel statistical genetic analyses focused on
transcriptomics, while also developing a novel strategy for genetic
manipulation of patient-derived epithelial cells. There are four major
biomedical genomics focus areas addressed by the research, namely fine mapping
of loci influencing inflammatory bowel disease, elucidation of the cell and
molecular function of causal genes, understanding how polymorphism influences
pathology, and translating quantitative genetic discoveries into clinical
outcomes. Specifically, integrative genomics expertise will be used to refine
the credible intervals responsible for complex association signals at
individual loci, enhance transcriptional risk scores (TRS) that have recently
been shown to provide much greater prediction of disease and progression than
genetic risk scores, and explore the potential of in silico predicted
transcriptome-wide association studies in the context of IBD. These studies
will utilize the IBDGC datasets through collaborative arrangements mediated by
data coordinating center. In addition, proof of principle for the use of lipid
nanoparticles as an efficient and specific delivery system for genome editing
and/or pharmaceutical delivery to targeted cell types in gut- derived
organoids will be demonstrated. Single cell RNA-Seq will be used to partition
variability in gut epithelial gene expression in the half dozen most common
organoid cell types into contributions of the ethnicity, location of the
biopsy, type of disease, and source laboratory. This data will serve as a
foundation for evaluating the effects of a half dozen gene knock-outs across
cell types using the lipid nanoparticle delivery system. All analyses and
reagents will be made available to consortium members as expected for
collaborative IBDGC research.


### Understanding how a loss of epigenetic reader SP140 contributes to IBD

- Kate Jeffrey, Massachusetts General Hospital
- Liaison: John Rioux

Genetics underlie susceptibility to inflammatory bowel disease (IBD) but the
rapid rise in incidence, as well as low concordance rates, point to a
prevalent role for the environment and possibly the epigenome. Dysregulation
of epigenetic enzymes is a recurrent and sentinel event in cancer, making
proteins that “write”, “erase” and “read” the epigenome some of the most
promising and intently pursued targets in drug discovery today. Despite this,
virtually nothing is known about how altered epigenetic enzymes contribute to
immunological disorders, including IBD. The long-term goal of the proposed
research is to determine how epigenetic alterations influence IBD and
accurately decipher the disease-causing mechanisms for better therapeutic
strategies. Speckled Protein 140 (SP140) is an epigenetic “reader” protein
with immune-restricted expression. Single nucleotide Polymorphisms (SNPs)
within SP140 associate with Crohn's disease (CD) (1, 2) however, the function
of SP140 and the consequences of SP140 SNPs have remained unclear. We found
CD- associated SP140 SNPs resulted in altered SP140 mRNA splicing and a loss
of SP140 protein (4). We also identified SP140 as a key orchestrator of
macrophage transcriptional programs essential for cellular identity and
function (4). Furthermore, we found that hematopoietic depletion of Sp140 in
mice exacerbated colitis and a loss of SP140 due to genetic variation
contributed to a subset of CD characterized by suppressed innate immunity (4).
The overall objective of the present proposal, which is the next step towards
attainment of our long-term goal, is to utilize multiple resources from the
IBD Genetics Consortium to precisely define the roles of SP140 in mediating
protective innate and adaptive immunity toward commensal microorganisms, and
determine how common or rare variants of SP140 impacts these ascribed
functions. Guided by compelling preliminary data, our hypotheses will be
tested in three specific aims: 1) Elucidate the role of SP140 in protective
innate immunity toward commensal microorganisms; 2) Define the role of SP140
in B cell development and function; and 3) Determine the mechanism of SP140 as
an epigenetic regulator. Under the first and second aims we will utilize our
inducible SP140 depletion system in mice and in parallel examine cells from
patients carrying SP140 variants. For aim 3 we will determine the precise
mechanism by which SP140 regulates immune cell identity and function. The
approach is innovative in the applicant's opinion because the work will be one
of the first descriptions of altered epigenetic enzyme expression and function
as a driver of immune-mediated disease. The proposed research is significant
because SP140 variants are associated with CD, MS and CLL so understanding the
function of SP140 will aid understanding of, and stratification of, these
diseases. Ultimately, such knowledge has the potential to vertically advance
development of novel therapeutics for CD and other immune disorders.



### Functional Molecular Investigation of Inflammatory Bowel Disease (IBD) Risk Variants

- Alexander Marson, University of California, San Francisco 
- Liaison: Rick Duerr

Inflammatory bowel disease (IBD) is a chronic disease characterized by
intermittent episodes of intestinal inflammation and disruption of the
intestinal epithelial barrier. The IBD Genetics Consortium has intensively
studied the genetic architecture of this complex disease. Assigning molecular
mechanisms to IBD risk variants is critical to understanding disease etiology
and identify new drug targets. Human genetics has potential to provide an
unbiased view of the causative disease mechanisms. IBD already has been the
subject of intensive genetic investigations, including genome-wide association
studies (GWAS) that have uncovered dozens of risk loci. However, mechanistic
understanding of these risk loci has been a challenge because the vast
majority falls outside of genes, in non-coding regions of the genome. In
contrast to protein-coding regions of the genome where we understand the amino
acid code, we still do not have a clear framework to understand how non-coding
genome variants alter cell function and contribute to disease. To learn how
DNA variation throughout the genome affects cellular pathways and contributes
to IBD, we now need a deeper understanding of the function of non-coding
genome elements in the specific cell types that drive the pathology. We
propose targeted CRISPR-based genome perturbations in primary human T cells,
human intestinal organoids (HIOs) and in vivo murine models of IBD pathology
to characterize both critical cis-regulatory elements and functional pathways
that are affected by IBD risk variants. Genome perturbations in primary immune
cells, HIOs and murine models will identify target genes regulated by these
critical non-coding elements and identify the cell-type or
stimulation-specific conditions where IBD risk variants impair gene
regulation. We will assess how non-coding variation at sites implicated by
human genetics alters gene regulatory programs in inflammatory and regulatory
T cells in both resting state and in response to stimuli. In addition, we will
investigate the contribution of specific genetic loci to intestinal
dysfunction in proliferation, epithelial barrier integrity, autophagy,
cellular stress responses and regenerative ability using gene-edited HIOs.
Understanding how causal non-coding IBD risk variants disrupt key gene
programs in human cells has potential to accelerate development of targeted
therapeutic approaches.


### Identifying population-specific IBD-associated mutations, genes and pathways

- Yuval Itan, Icahn School of Medicine at Mount Sinai

Inflammatory bowel disease (IBD) is a group of disorders that involve chronic
inflammation of the colon and small intestine. The two major types of IBD are
ulcerative colitis (UC): long term inflammation and ulcers of the colon and
rectum, and Crohn’s Disease (CD): inflammation of the digestive tract lining
that can spread into affected tissues. IBD can be debilitating, and life
threatening in severe cases. There are currently over 1.5 million estimated
IBD patients in the United Stated. IBD results in substantial direct
healthcare costs (over US$6 billion per year in the United States) and the
incidence of IBD continues to rise worldwide. Over the last two decades
genetics has been shown to be a major contributor to IBD predisposition. Most
studies investigating IBD have focused on genome-wide association studies
(GWAS) of common genetic variants, mostly in European populations. While it
has been shown that IBD has a substantial Mendelian/monogenic component of
high impact mutations, and that it is likely that a significant proportion of
IBD mutations and genes are population-specific (for example: occur in African
Americans and not in Europeans), there has not yet been an effort to perform
in IBD (or in any other disease) a large-scale investigation of
population-specific IBD-causing high-impact mutations, genes and pathways.
Understanding population-specific IBD genomics would ultimately contribute to
personalized medicine of IBD patients, where the population background of the
patient will be crossed with the patient’s estimated IBD mutations to tailor
an optimal therapy. We hypothesize that IBD has a major component of
population-specific high-impact predisposing mutations and genes, and that
these mutations and genes can be discovered with gene burden analyses on whole
exome sequencing (WES) data of IBD patients. We therefore propose to perform
an ancillary study on IBDGC’s WES data of IBD patients that consists of four
major human populations: Ashkenazi Jewish, African Americans, Hispanics and
Europeans. IBDGC is ideal for such a study, as it contains enough samples per
population group to perform WES gene burden analyses of high-impact genetic
variants (shown in preliminary results). We also have full access to Mount
Sinai’s BioMe BioBank, which contains WES data of over 30,845 patients across
diverse populations (the majority being the four populations of IBDGC)
including IBD patients, that will boost our sample sizes and analysis power.
We propose a rigorous analysis pipeline combining state-of-the-art established
software with cutting-edge approaches across human population and disease
genomics, as well as diverse computational algorithms to: (1) obtain a
high-quality set of IBD cases and controls by genetic population estimates of
Ashkenazi Jewish, African Americans, Hispanics and Europeans on IBDGC data;
(2) performing a gene burden cases-controls analysis for high impact variants
in each population; and (3) identifying population- specific IBD-associated
mutations, genes and pathways, performing computational functional relatedness
of the results to IBD, and performing phenome-wide association (PheWas) to IBD
on Mount Sinai’s BioMe BioBank.


## Current R21 Projects

### Role of Cytomegalovirus in the Natural History of Crohn’s Disease

- Ravit Boger, Johns Hopkins University School of Medicine
- Liaison: Steve Brant

Crohn's disease (CD) is a multifactorial and complex disease, orchestrated by
several genetic and environmental triggers. The best known genetic marker for
CD is the nucleotide oligomerization domain 2 (NOD2), a cytoplasmic receptor
that responds to bacterial pathogens. Mutations in NOD2 are the strongest
genetic marker for CD and the only proven risk factor for stricturing
phenotypes. The fact that mutations in NOD2 result in CD only in some
individuals but not in others indicates that environmental factors may
contribute to or trigger CD. CMV, a common viral infection, establishes
life-long latency in the host including in intestinal epithelial cells. Our
laboratory reported that NOD2 suppresses CMV replication, but the NOD2 mutant
3020insC, associated with severe CD, is unable to inhibit CMV replication in
vitro. NOD2 activation with muramyl dipeptide (MDP) inhibits CMV replication
in vitro. Furthermore, MDP treatment of mouse CMV (MCMV) infected macrophages
results in virus suppression. In contrast, in macrophages collected from
NOD2-deficient mice MDP cannot suppress virus replication. These data provide
strong support for the role of CMV as a modifier of NOD2 innate immune
responses and CD pathogenesis. We hypothesize that CMV infection contributes
to the natural history and complications of CD, especially in genetically
prone individuals who carry NOD2 mutations. We will use the NIDDK IBD Genetics
Consortium (IBDGC) tissue bank and corresponding data from their “Ileal
Post-Operative Recurrence Study” of CD patients that have undergone ileal
resection for any indication. We will determine whether CMV is more readily
detected in tissue samples from patients with NOD2 mutations. Analysis will be
controlled for drugs such as steroids and other immunosuppressive agents. In
cases of CMV detection in tissue samples we will quantify CMV DNA in blood.
The contribution of CMV to stricturing disease behavior and to early
post-operative recurrences will also be determined in tissue samples. Our
study will provide the first detailed evaluation of CMV as a risk factor for
CD stricturing complications in the context of a specific genetic background.
Understanding the interplay between genetic and infectious triggers of CD may
lead to the development of biomarkers for disease sequalae and inform future
therapeutic strategies. Data from our studies could have major clinical
implications for patients with CD, as individuals with known NOD2 loss of
function mutations may benefit from early diagnosis of CMV, prophylaxis or
anti-viral therapy.


### Profiling intraepithelial lymphocyte populations in health and Crohn's disease

- Karen Edelblum, RBHS, New Jersey Medical School
- Liaison: Steve Brant

Immunological surveillance at barrier surfaces is essential to provide defense
against enteric pathogens and maintain mucosal homeostasis. Disruption of the
balance between pro-inflammatory and regulatory immune response can lead to a
loss of mucosal tolerance and development of chronic inflammatory disease,
such as Crohn's disease (CD). The majority of inflammatory responses in CD
have been attributed to both innate and adaptive immune cells located in the
lamina propria; however, relatively little is known regarding the contribution
of the intraepithelial lymphocyte (IEL) compartment to CD pathogenesis. IELs,
which include intraepithelial T cells (IET) and intraepithelial ILCs (IE-ILC),
contribute to the maintenance of epithelial barrier integrity and function as
first line of defense against luminal microorganisms. Due to the difficulty in
obtaining a sufficient number of IELs from patient biopsies for in-depth
functional analyses, we propose to leverage advances in single-cell RNA
sequencing technology and RNA imaging to profile the phenotypic and functional
characteristics of ileal IEL subpopulations in health and acute CD. Using
transcriptomics as a guide to identify molecular signatures for individual IEL
subsets, we will develop single-molecule fluorescence in situ hybridization
(sm-FISH) probe multiplexes to visualize these cell populations in patient
biopsies. In collaboration with the IBD Genetics Consortium, we will obtain
tissue sections from longitudinal endoscopic biopsies of a CD patient cohort
that has undergone ileal resection and determine the extent to which IEL
phenotype, localization and effector function correlate to disease
pathogenesis. These experiments will not only provide the first cell atlas of
the human ileal IEL compartment, but may also lead to the development of novel
prognostic biomarkers or therapeutic interventions for the treatment of CD.


### Leveraging the epigenome of inflammatory bowel disease to gain mechanistic insights into disease pathophysiology

- Subra Kugathasan, Emory University
- Liaison: Judy Cho

Ulcerative colitis (UC) is a chronic relapsing and remitting intestinal
inflammatory disorder with a very heterogeneous clinical course. While on
life-long maintenance therapy, although most patients achieve complete mucosal
healing with no disease activity during the course of treatment of UC, a
subgroup (~40%) experience chronically active severe disease with persistent
inflammation as reflected by need for escalation of medical therapy or
surgery. The reasons underlying such differential clinical course/disease
severity are not well understood. Cross-sectional studies of DNA methylation,
a key regulator of gene expression and molecular phenotype, have begun to
reveal epigenetic associations with UC. However, owing to the dynamic
plasticity of DNA methylation and UC disease behavior, it is critical to
understand the temporal relationship between the methylome and the disease in
order to establish the direction of causality and leverage the epigenome for
therapeutic benefits. Here we hypothesize that, longitudinal framework –
having DNA methylation data and well documented disease measures collected
concurrently during the disease onset and at later time – supplemented by
genetic association and the concept of Mendelian randomization, can help
identify methylation changes that causally underlie disease pathology. Herein,
using methylation data generated from DNA derived from the disease-relevant
tissue, rectal mucosa, obtained at two time points – at diagnosis and 1 year
follow-up – from participants in the PROTECT cohort, a pediatric prospective
inception UC cohort, we plan to (i) identify DNA methylation changes that
causally influence the development of UC, and modify its phenotypic expression
and severity; and (ii) integrate these methylation data with prior genotype
and gene expression data in order to elucidate the functional consequence of
disrupted methylation patterns and to gain insights into molecular
underpinnings of UC. In completion, the results of this project will provide
new insights into the epigenetic basis of UC. Understanding the temporal
relationship between how methylome changes during the course of the disease,
as a result of varying clinical characteristics, and how disease subgroups
evolve may aid in the identification of potentially causal epigenetic targets
which could subsequently be leveraged for therapeutic benefits.
