+++
widget = "blank"
headless = true
weight = 30
title = "History"
subtitle = ""

[design]
columns = "2"

  [design.background]
  color = "#6381A6"

  text_color_light = true

  [design.spacing]
  padding = [ "", "", "", "50px" ]

+++

The NIDDK IBDGC was established in 2003 based on the recognition that the most effective way to advance mechanistic understanding of a common, complex genetic disease such as IBD in a rigorous, well-powered manner would be via a cooperative agreement between major IBD Centers across North America. Comprised of six Genetics Research Centers (GRCs) and a single Data Coordinating Center (DCC), the IBDGC is currently in its fourth five-year funding period. In the first period, development of common data collection instruments and recruitment of IBD patients and controls into the NIDDK lymphoblastoid cell line and phenotype data repository were begun. The NIDDK IBDGC was fortuitously started shortly after the completion the first phase of the Human Genome Project (~2001) and discovery of the first genes for ulcerative colitis (MHC class II region) and Crohn’s disease (NOD2). The HapMap Project enabled development of high quality, SNP-based chips that comprehensively and accurately sampled common variation across populations. These chips continue to form a highly cost-effective, efficient, and comprehensive means of assaying most genetic variation contributing to IBD.

The IBDGC focused on GWAS-based studies during the second and third periods of funding, identifying over 260 loci by 2017. The IBD genetics field has enjoyed extraordinary international cooperation revolving around the International IBD Genetics Consortium (IIBDGC) for which the NIDDK Consortium has played a leadership role. The NIDDK IBDGC DCC serves as the sole data coordinating center for the IIBDGC, with major efforts ongoing for both chip- and sequence-based efforts. While we have recruited African-American and Hispanic patients since the beginning of the NIDDK IBDGC, it was only in 2021 that the first genome-wide significant locus in African-American Crohn’s disease (chromosome 5p13, near PTGER4) was reported. It is axiomatic in complex genetics that credibly causal genetic variants should be shared across populations. In IBD, while there is general concordance of loci across populations, it is striking in Crohn’s disease that the largest effect loci are either unique to a population (e.g., NOD2 variants not present in Asian populations) or have substantially different effect sizes (e.g., chr 9q32 near TNFSF15 with largest effect in Asian populations). While studies in African populations have not yielded substantial common variant locus discoveries thus far, greater protein-altering genetic diversity and shorter linkage disequilibrium blocks may assist in defining credibly causal genes and cells, respectively.

During the third and present periods of funding, numerous GRC-based pilot efforts have been supported. Longitudinal studies at key clinical inflection points have been initiated and are ongoing, most prominently with respect to the post-operative ileal resection period for Crohn’s disease. Given the substantial enrichment of GWAS credible SNP lists in transcriptionally active epithelial regions, especially for ulcerative colitis, investments in multi-omic analyses at the single cell level have been made. The vast majority of IBD loci are shared between Crohn’s disease and ulcerative colitis, but the differences in loci and lead SNPs provides a substantial opportunity for exploring, at the single cell level, fundamental pathophysiologic differences between these two major sub-types. By investing in well-powered, multi-site studies reflecting clinical heterogeneity (e.g., ileal vs. rectal/perianal Crohn’s disease), key inflection points, and unmet medical needs, the NIDDK IBDGC will continue to drive and evolve efforts to advance mechanistic understanding to improve patient outcomes.
