---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "Functional Molecular Investigation of Inflammatory Bowel Disease (IBD) Risk Variants"
summary: ""
authors: ["Alexander Marson", "Liaison: Rick Duerr"]
tags: [R01, Ancillary]
categories: [R01]
date: 2021-03-12T16:56:38-06:00

# Optional external URL for project (replaces project detail page).
external_link: ""

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: true

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""
---

Inflammatory bowel disease (IBD) is a chronic disease characterized by intermittent episodes of intestinal inflammation and disruption of the intestinal epithelial barrier. The IBD Genetics Consortium has intensively studied the genetic architecture of this complex disease. Assigning molecular mechanisms to IBD risk variants is critical to understanding disease etiology and identify new drug targets. Human genetics has potential to provide an unbiased view of the causative disease mechanisms. IBD already has been the subject of intensive genetic investigations, including genome-wide association studies (GWAS) that have uncovered dozens of risk loci. However, mechanistic understanding of these risk loci has been a challenge because the vast majority falls outside of genes, in non-coding regions of the genome. In contrast to protein-coding regions of the genome where we understand the amino acid code, we still do not have a clear framework to understand how non-coding genome variants alter cell function and contribute to disease. To learn how DNA variation throughout the genome affects cellular pathways and contributes to IBD, we now need a deeper understanding of the function of non-coding genome elements in the specific cell types that drive the pathology. We propose targeted CRISPR-based genome perturbations in primary human T cells, human intestinal organoids (HIOs) and in vivo murine models of IBD pathology to characterize both critical cis-regulatory elements and functional pathways that are affected by IBD risk variants. Genome perturbations in primary immune cells, HIOs and murine models will identify target genes regulated by these critical non-coding elements and identify the cell-type or stimulation-specific conditions where IBD risk variants impair gene regulation. We will assess how non-coding variation at sites implicated by human genetics alters gene regulatory programs in inflammatory and regulatory T cells in both resting state and in response to stimuli. In addition, we will investigate the contribution of specific genetic loci to intestinal dysfunction in proliferation, epithelial barrier integrity, autophagy, cellular stress responses and regenerative ability using gene-edited HIOs. Understanding how causal non-coding IBD risk variants disrupt key gene programs in human cells has potential to accelerate development of targeted therapeutic approaches.
