---
title: "Dynamics of metatranscription in the inflammatory bowel disease gut microbiome"
date: 2018-01-01
publishDate: 2020-08-03T19:38:06.598514Z
authors: ["Melanie Schirmer", "Eric A. Franzosa", "Jason Lloyd-Price", "Lauren J. McIver", "Randall Schwager", "Tiffany W. Poon", "Ashwin N. Ananthakrishnan", "Elizabeth Andrews", "Gildardo Barron", "Kathleen Lake", "Mahadev Prasad", "Jenny Sauk", "Betsy Stevens", "Robin G. Wilson", "Jonathan Braun", "Lee A. Denson", "Subra Kugathasan", "Dermot P. B. McGovern", "Hera Vlamakis", "Ramnik J. Xavier", "Curtis Huttenhower"]
publication_types: ["2"]
abstract: "Inflammatory bowel disease (IBD) is a group of chronic diseases of the digestive tract that affects millions of people worldwide. Genetic, environmental and microbial factors have been implicated in the onset and exacerbation of IBD. However, the mechanisms associating gut microbial dysbioses and aberrant immune responses remain largely unknown. The integrative Human Microbiome Project seeks to close these gaps by examining the dynamics of microbiome functionality in disease by profiling the gut microbiomes of textgreater100 individuals sampled over a 1-year period. Here, we present the first results based on 78 paired faecal metagenomes and metatranscriptomes, and 222 additional metagenomes from 59 patients with Crohn's disease, 34 with ulcerative colitis and 24 non-IBD control patients. We demonstrate several cases in which measures of microbial gene expression in the inflamed gut can be informative relative to metagenomic profiles of functional potential. First, although many microbial organisms exhibited concordant DNA and RNA abundances, we also detected species-specific biases in transcriptional activity, revealing predominant transcription of pathways by individual microorganisms per host (for example, by Faecalibacterium prausnitzii). Thus, a loss of these organisms in disease may have more far-reaching consequences than suggested by their genomic abundances. Furthermore, we identified organisms that were metagenomically abundant but inactive or dormant in the gut with little or no expression (for example, Dialister invisus). Last, certain disease-specific microbial characteristics were more pronounced or only detectable at the transcript level, such as pathways that were predominantly expressed by different organisms in patients with IBD (for example, Bacteroides vulgatus and Alistipes putredinis). This provides potential insights into gut microbial pathway transcription that can vary over time, inducing phenotypical changes that are complementary to those linked to metagenomic abundances. The study's results highlight the strength of analysing both the activity and the presence of gut microorganisms to provide insight into the role of the microbiome in IBD."
featured: false
publication: "*Nature Microbiology*"
tags: ["Female", "Humans", "Male", "Adult", "Young Adult", "Adolescent", "Longitudinal Studies", "Child", "Colitis", "Ulcerative", "Crohn Disease", "Inflammatory Bowel Diseases", "Gastrointestinal Microbiome", "Phenotype", "Dysbiosis", "Feces", "Gene Expression Profiling", "Metagenomics", "Transcription", "Genetic"]
doi: "10.1038/s41564-017-0089-z"
---

