---
# Adapted from themes/academic/archetypes/project/index.md
title: "Data Coordinating Center"
subtitle: "Icahn School of Medicine at Mount Sinai and the University of Chicago"
pi: "Judy Cho, MD"
pi_titles: ["Principal Investigator, IBDGC Data Coordinating Center"]
picture: "/img/mssm/judy-cho-pic.jpg"
bio: ["My major research efforts currently are: a) defining mechanisms of stromal/epithelial-immune cell cross talk in IBD; b) defining components of treatment refractory (specifically, non-response to pro-inflammatory cytokine blockade) IBD at the cellular and tissue level; c) predicting differences in clinical outcomes of interest across IBD clinical subtypes and populations; d) exploring the role of rare, IBD-associated variants of high effects; e) organizing large datasets for NIDDK IBDGC and IIBDGC; and f) developing select treatment targets based on IBD genetic and single cell expression findings. Since 2015, I have served as Director of the Personalized Medicine Institute at Mount Sinai. I am extremely excited by the explosion of data and new data tools that are accelerating collaborative efforts to benefit patients through genetic and genomic studies; I am eager to continue applying new data-approaches to benefit the NIDDK IBDGC."]
tags: []
categories: []
date: 2020-08-07T09:46:16-04:00

# Optional external URL for PI
external_link: "https://icahn.mssm.edu/profiles/judy-h-cho"

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.

publications:
    count: 12
    order: "desc"
---

# DCC Objectives

IBD affects 1.8 to 3 million Americans and better therapies are urgently
needed. The IBDGC, comprised of the DCC and six Genetic Research Centers
(GRCs) throughout North America, has played a central role in catalyzing
genetic progress in IBD. During the past funding cycle, the IBDGC has
continued to leverage the large, cross-national datasets it has compiled to
push to 200 the number of genetic loci associated with IBD risk. In addition,
it has begun the work of fine-mapping these loci and conducting the first-ever
large-scale genetic studies among non-European populations, both critical next
steps in further developing our understanding of the overall genetic
architecture of IBD. The IBDGC has also initiated two new studies—one in CD
and one in UC—involving novel protocols for intestinal sampling and
longitudinal follow-up including serial samples of blood and stool. These
studies will enable the IBDGC to begin to focus on the precise molecular
mechanisms responsible for genetic variation in IBD risk, as well as to
identify the specific cells in which disease-associated variants exert their
pathogenic or protective effects. This knowledge will be critical in more
accurately prioritizing which pathways to target for the development of new
therapies for IBD.

To continue this progress, the DCC will focus over the next 5 years on two
related fronts:

1. The DCC will apply new laboratory and computing technologies to collect
   accurate, detailed data on disease pathogenesis from the clinical all the
   way down to the cellular level, and to integrate these data with both
   existing and new genomic data for sharing with the research community.

   1. Continue to develop our platform for collecting detailed, longitudinal
      phenotypic and clinical data, and apply it to larger cohorts and in
      targeted ways (e.g., to follow individuals with rare genetic variants or
      unique phenotypic characteristics)
   2. Increase collaboration with the top laboratories doing functional studies
      in IBD, providing them with biological material and linking their results
      to genetics
   3. Develop new objective assessments of disease activity and treatment that
      can be efficiently applied at scale, including automated methods for
      extracting information from endoscopy and pathology reports, virtual
      slides of endoscopic biopsies, endoscopic video and Electronic Medical
      Records (EMRs)
   4. Use the latest cloud computing technologies to facilitate collaborative
      analysis through the Bionimbus Protected Data Cloud, and facilitate data
      sharing by developing an IBD Data Commons

2. The DCC will prioritize scientific directions for the Consortium and scale
   up new technologies, sampling strategies and analyses by applying them to
   larger cohorts and longitudinal data.

   1. Enhance fine-mapping precision of established loci by expanding sample
      size and diversity. We will extend methodologies used to distinguish
      credible SNPs between CD and UC to include other inflammatory diseases
      and non-European ancestry populations to improve fine-mapping scope and
      power.
   2. Comprehensively define the genetic basis for altered intestinal
      epithelial gene expression and function in UC. We will analyze sources
      of variation in mRNA expression from spheroids grown from biopsies taken
      from inflamed and uninflamed UC colon.
   3. Determine the genetic basis for how pathophysiologic mechanisms function
      over time by studying variation in response to therapeutic
      interventions, and by modeling the disease process longitudinally. We
      have developed protocols for longitudinal analyses of gene expression,
      metabolome and microbiome following ileal resection in CD. We propose to
      continue recruitment beyond our current collections of over 300 patients
      and extend recruitment to other centers beyond the Consortium.
