---
# Adapted from themes/academic/archetypes/project/index.md
title: "Data Coordinating Center"
subtitle: "Icahn School of Medicine at Mount Sinai and the University of Chicago"
pi: "Judy Cho, MD"
pi_titles: ["Principal Investigator, IBDGC Data Coordinating Center", "Professor, Icahn School of Medicaine at Mount Sinai"]
picture: "/img/mssm/judy-cho-pic.jpg"
bio: ["Dr. Cho has served as the Principal Investigator of the NIDDK IBDGC since 2003. She plays a lead role in setting scientific and operational priorities, assessing new sample and analytic approaches, and overseeing internal and external communication."]
tags: []
categories: []
date: 2020-08-07T09:46:16-04:00

# Optional external URL for PI
external_link: "https://icahn.mssm.edu/profiles/judy-h-cho"

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.

publications:
    count: 12
    order: "desc"
---

# DCC Objectives

<<<<<<< HEAD
The Data Coordinating Center (DCC) of the NIDDK IBDGC plays a lead role in a) assessing and applying innovative research approaches in genetics and genomics to define pathophysiologic mechanisms, b) operationally unifying the NIDDK IBDGC and assuring rigor and reproducibility at scale, and c) facilitating collaboration with external investigators and groups to maximize the size and scientific value of existing datasets, provide complementary expertise, and develop the next generation of IBD investigators pursuing genetic, cellular and molecular understanding of IBD.
=======
The Data Coordinating Center of the NIDDK IBDGC plays a lead role in a) assessing and applying innovative research approaches in genetics and genomics to define pathophysiologic mechanisms, b) operationally unifying the NIDDK IBDGC, assuring rigor and reproducibility, and appropriate scale for the proposed work, and c) serving a key facilitative role for external investigators and groups, to increase scale, provide complementary expertise and develop the next generation of IBD investigators possessing genetic, cellular and molecular understanding of IBD.
>>>>>>> eb9aba46a146eb81809b8f0d97fc76529b4e4cac

---

# Specific Aims: 2017-2022

**Aim 1:** To apply new laboratory and computing technologies and collect accurate, detailed data on disease pathogenesis from the clinical all the way down to the cellular level and genetic level, sharing with the broader research community.
- Expand platforms for collecting detailed, longitudinal cohort collections
- Efficiently collaborate with top laboratories, operationalizing sample and data flows and linking results to genetics
- Develop and scale objective assessments of disease activity and treatment including automated methods for extracting information from endoscopy and pathology reports, virtual slides of endoscopic biopsies, endoscopic video and Electronic Medical Records (EMRs)
- Develop and apply a cloud-based IBD Data Commons, for broad investigator use

**Aim 2:** To prioritize scientific directions for the Consortium and scale new technologies, sampling strategies and analyses to larger cohorts, including longitudinal data.
- To fine-map established loci by expanding sample size and diversity. To distinguish credible SNPs between CD and UC
- Comprehensively define the genetic basis for altered intestinal epithelial gene expression and function in UC. We will analyze sources of variation in mRNA expression from spheroids grown from biopsies taken from inflamed and uninflamed UC colon.
- Determine the genetic basis for how pathophysiologic mechanisms function over time, including in response to therapeutic interventions

---

<<<<<<< HEAD
# Ongoing and New Directions

### New recruitment priorities and approaches

The DCC is piloting methods to facilitate recruitment in non-European populations using single IRBs, alternative reimbursement strategies, biobank-based and direct-to-patient recruitment approaches. The DCC leads a monthly coordinators’ meeting to communicate IBDGC priorities, assess progress and share effective recruiting and sample handling approaches across the Consortium.

### New data types and accelerating integration

The DCC is adding to its form-based methods for collecting phenotype data by using electronic medical records and diagnostic imaging and reports. IBDGC investigators are piloting image-based measurements beginning with radiology but likely expanding into both pathology and paraffin-based methods as well. Rapidly developing applications of multi-omic single cell technologies are a high priority, especially for their ability to provide insight into IBD physiology.

### Data sharing, data mining, and project acceleration

Common variant, GWAS-based data mapped to traits represents an important, foundation for further studies. Expansions of both chip and sequence-based cohorts will be finalized in 2021-2022 and published via dbGaP and the IBDGC Data Commons. The integration of African-based reference, common variant data substantially improves IBD trait prediction as an important basis for the expansion of non-European cohorts. Tissue-based, bulk RNAseq gene expression linked to important clinical inflection points has provided substantial mechanistic insight, especially when integrated with single cell data. Through the DCC, the IBDGC will utilize the Data Commons to increase the speed of analyses and data sharing in promotion of deeper collaboration.

### Training the next generation of IBD researchers

The DCC organizes a rotating schedule of GRC presentations where work from junior investigators is encouraged. In 2020, the DCC initiated a webinar series focusing on data-intensive, translational investigators. Given a) the rapid rate of data expansion in scope and type, and b) the increasingly refined mining capacities of extant clinical collections, the opportunities for training and developing the next generation of translational researchers are highly promising.
=======
# Ongoing and New Directions for the DCC

1. Recruitment priorities and approaches. The NIDDK IBDGC steering committee has prioritized the collection of non-European IBD cases, continuing expansion of blood- and tissue-based cohorts. The DCC is piloting new approaches to scale recruitment using single IRBs, alternative re-imbursement flows, biobank-based and other non-GRC collaborations and direct-to-patient recruitment approaches. The DCC has instituted a monthly coordinators’ meeting to communicate IBDGC priorities, assess progress and share effective recruiting and sample handling approaches across the IBDGC.

2. New data-types and accelerating data integration: beyond locus discovery. Beyond the traditional form-based phenotype collections (RedCap), the DCC has instituted collections of endoscopic, radiologic and pathologic images and reports. IBDGC investigators are piloting more objective, direct image-based measurements, beginning with radiology, but likely expanding substantially in both pathology images and paraffin-based new technologies. Rapidly developing applications of multi-omic single cell technologies are a high priority, especially in providing insight into blood-to-tissue IBD physiology.

3. Data sharing, data mining and project acceleration. Common variant, GWAS-based data mapped to traits (Crohn’s disease, ulcerative colitis, IBD) in primarily European ancestry cohorts represents an important, well-powered foundation for further studies. Major expansions of both chip-based and sequence-based cohorts will be finalized for broad-sharing in 2021-2022. The extent to which uncommon and extremely rare single variants contribute to trait variance will be empirically determined. That integration of African-based reference, common variant case-control data so substantially improves IBD trait prediction forms one important basis for the proposed, major expansion of non-European ancestry cohorts. Tissue-based, bulk RNASeq gene expression linked to important clinical inflection points (disease onset, serially in response to new treatments) has provided substantial mechanistic insight, especially when integrated with single cell data. Genetic variants act through heterotypic cells, where integration with contextual, multi-omic single cell data across populations will be appliied. Both NIDDK IBDGC-generated and other data sources can will be applied. While peer-reviewed manuscripts continue be the most generally useful means of data sharing and dissemination, the DCC will pilot earlier data-sharing through new collaborations, sharing data, samples and code broadly to accelerate progress.

4. Training the next generation of genetic, cellular and molecular mechanism-based IBD translational researchers. The DCC organizes a rotating schedule of GRC presentations, where work from more junior investigators in the GRC teams is encouraged. In 2020, the DCC instituted a Webinar series, focused on data-intensive, translational investigators. The mailing list included Principal Investigators from all of the NIDDK-supported Digestive Disease Centers. For each of seminars, over 140 participants attended, reflecting substantial interest. Given a) the datasets rapidly expanding in scope and type, and b) increasingly refined mining capacities of extant clinical collections, the opportunities for training and developing the next generation of translational researchers are highly promising.
>>>>>>> eb9aba46a146eb81809b8f0d97fc76529b4e4cac
