---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Prioritizing Crohnâ€™s disease genes by integrating association signals with
  gene expression implicates monocyte subsets
subtitle: ''
summary: ''
authors:
- Kyle Gettler
- Mamta Giri
- Ephraim Kenigsberg
- Jerome Martin
- Ling-Shiang Chuang
- Nai-Yun Hsu
- Lee A. Denson
- Jeffrey S. Hyams
- Anne Griffiths
- Joshua D. Noe
- Wallace V. Crandall
- David R. Mack
- Richard Kellermayer
- Clara Abraham
- Gabriel Hoffman
- Subra Kugathasan
- Judy H. Cho
tags: []
categories: []
date: '2019-01-01'
lastmod: 2021-11-01T08:41:35-04:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-11-01T12:41:34.985671Z'
publication_types:
- '2'
abstract: Genome-wide association studies have identified ~170 loci associated with
  Crohn's disease (CD) and defining which genes drive these association signals is
  a major challenge. The primary aim of this study was to define which CD locus genes
  are most likely to be disease related. We developed a gene prioritization regression
  model (GPRM) by integrating complementary mRNA expression datasets, including bulk
  RNA-Seq from the terminal ileum of 302 newly diagnosed, untreated CD patients and
  controls, and in stimulated monocytes. Transcriptome-wide association and co-expression
  network analyses were performed on the ileal RNA-Seq datasets, identifying 40 genome-wide
  significant genes. Co-expression network analysis identified a single gene module,
  which was substantially enriched for CD locus genes and most highly expressed in
  monocytes. By including expression-based and epigenetic information, we refined
  likely CD genes to 2.5 prioritized genes per locus from an average of 7.8 total
  genes. We validated our model structure using cross-validation and our prioritization
  results by protein-association network analyses, which demonstrated significantly
  higher CD gene interactions for prioritized compared with non-prioritized genes.
  Although individual datasets cannot convey all of the information relevant to a
  disease, combining data from multiple relevant expression-based datasets improves
  prediction of disease genes and helps to further understanding of disease pathogenesis.
publication: '*Genes and Immunity*'
doi: 10.1038/s41435-019-0059-y
---
