---
# Adapted from themes/academic/archetypes/project/index.md
title: "IBD Genetics Research Center at the University of Pittsburgh"
subtitle: ""
pi: "Richard H. Duerr, MD"
pi_titles: ["Inflammatory Bowel Disease Genetic Research Endowed Chair Professor of Medicine, Human Genetics, and Clinical and Translational Science, University of Pittsburgh", "Co-Director and Scientific Director, Inflammatory Bowel Disease Center UPMC"]
picture: "/img/pittsburgh/richard-duerr.jpg"
bio: ["Richard H. Duerr, MD, is the Principal Investigator of the NIDDK IBDGC’s Genetic Research Center at the University of Pittsburgh, where he holds the endowed Inflammatory Bowel Disease Genetic Research Chair and is Professor of Medicine, Human Genetics, and Clinical and Translational Science.  He is the Co-Director and Scientific Director of the University of Pittsburgh Medical Center (UPMC) Inflammatory Bowel Disease Center, and he serves as Associate Chief Scientist, Translational Research on the Leadership Team of the Crohn’s & Colitis Foundation's IBD Plexus research and information exchange platform.  Dr. Duerr’s research career began with an IBD-focused research and clinical fellowship at UCLA, where he received training in cellular immunology and also participated in studies that established the association between the serum biomarker, p-ANCA, and IBD.  He refocused his research efforts on the genetics of IBD when he established an independent laboratory at the University of Pittsburgh.  Dr. Duerr currently applies his expertise in human genetics, genomics, and immunology to define the molecular effects of IBD-associated, non-coding, accessible chromatin region DNA variation on human CD4+CD45RO+CD196+ T cell functional responses to stimulation, and to conduct high dimensional immunophenotyping and transcriptional profiling of immunocytes isolated from peripheral blood and from inflamed and non-inflamed intestinal mucosa using the cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) assay."]
tags: []
categories: []
date: 2020-08-07T09:46:16-04:00

# Optional external URL for PI
external_link: "https://profiles.dom.pitt.edu/gi/faculty_info.aspx/Duerr5020"

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.

publications:
    count: 12
    order: "desc"
---


# Co-investigators

- Wei Chen, PhD, Associate Professor of Pediatrics, Biostatistics, and Human
  Genetics, University of Pittsburgh


# Lead Clinicians

- Arthur Barrie, MD, PhD, Assistant Professor of Medicine, University of Pittsburgh
- David Binion, MD, Professor of Medicine, University of Pittsburgh; Co-Director and Translational Research Director, Inflammatory Bowel Disease Center, and Director, Nutrition Support Service, UPMC
- James Celebrezze, MD, FACS, FACRS, Assistant Professor of Surgery, University of Pittsburgh; Director of Colorectal Surgical Training, UPMC
- Jon Davison, MD, Associate Professor of Pathology, University of Pittsburgh
- Jeffrey Dueker, MD, MPH, Assistant Professor of Medicine, University of Pittsburgh
- Janet Harrison, MD, Assistant Professor of Medicine, University of Pittsburgh
- Jennifer Holder-Murray, MD, FACS, Vice Chair for Quality Integration and Assistant Professor of Surgery, University of Pittsburgh; Co-Director of Enhanced Recovery Program, UPMC
- Elyse Johnston, MD, Clinical Instructor of Medicine, University of Pittsburgh
- Kenneth Lee, MD, Jane and Carl Citron Chair in Colon Cancer, Vice Chair for Graduate Medical Education and Professor of Surgery, University of Pittsburgh; Program Director, General Surgery Residency Program, UPMC; and Adjunct Assistant Professor, Chatham College
- David Medich, MD, FACS, FASCRS, Chief, Division of Colon and Rectal Surgery and Associate Professor of Surgery, University of Pittsburgh
- Marc Schwartz, MD, Assistant Professor of Medicine, University of Pittsburgh
- Andrew Watson, MD, M. Litt, FACS, Assistant Professor of Surgery, University of Pittsburgh; Vice- President, Clinical Information Technology Transformation, International Division, and Director, Telemedicine, UPMC

# NIDDK IBDGC-related funding

Crohn’s disease and ulcerative colitis, the two major forms of idiopathic
inflammatory bowel disease (IBD), are chronic disorders of the
gastrointestinal tract affecting ~3.7 million Americans and Europeans, and
several hundred thousand more worldwide. Intensive searches for genetic risk
factors, including genome-wide association studies (GWAS) and follow-up
studies by the NIDDK Inflammatory Bowel Disease Genetics Consortium and the
International Inflammatory Bowel Disease Genetics Consortium have yielded
significant evidence for association between IBD and >240 genetic loci, and
have illuminated biological pathways such as the IL-23/T helper 17 (Th17)
immune pathway, autophagy, mucosal barrier function, and immune activation of
multiple integrin genes in IBD pathogenesis. While fine genetic association
mapping efforts have resolved 51 association signals to a single genetic
variant with >50% probability of being causal, the remaining known IBD loci
have yet to be fine mapped. Access to large patient sample sizes and the
multidisciplinary team science that a consortium facilitates are crucial for
the momentum in dissecting the genetics of IBD to continue, and for bridging
the gaps between genetic associations, mechanistic understanding, and improved
therapies.

In Aim 1 of this competitive renewal application we propose to continue our
participation in IBDGC research efforts, and we propose specific
consortium-wide projects for consideration by the Steering Committee. A major
challenge in moving from genetic associations to mechanistic understanding is
the fact that SNP(s) with the strongest association signal(s) in most
established IBD loci lie in non-coding regions, where our knowledge of the
functional effects of DNA variation pales in contrast to our understanding of
the genetic code for amino acids in protein-coding regions. Additionally,
genetic studies alone cannot distinguish causal variants from neighboring
non-causal variants in linkage disequilibrium for many known IBD association
signals. Fortunately, recent studies suggest a way forward. Comparing the
locations of non-coding, disease-associated SNPs with chromatin maps in
specific cell types and stimulation conditions can prioritize candidate
non-coding DNA variants, specific cell types and contexts for follow-up
functional studies.

In Aim 2, we will define the molecular effects of IBD-associated, non-coding,
accessible chromatin region DNA variation on primary human
CD4+CD45RO+CD196(CCR6)+ T cell (a subset including both Th17 and T regulatory
[Treg] cells) functional responses to specific stimulation conditions. This
work will build on our mapping of accessible chromatin regions in
CD4+CD45RO+CD196+ T cells stimulated under two specific conditions, our
identification of overlap between these accessible chromatin regions and SNPs
in the credible set for each fine mapped IBD locus that is >95% likely to
contain the causal variant, and our preliminary integration of chromatin
accessibility, RNA-seq and genotype data in the same samples. CRISPR/cas9
genome editing in primary human T cells will be utilized to test the
functional effects of accessible chromatin regions harboring IBD-associated
SNPs.

## Pilot Project on Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) in Colonoscopic Biopsy-Derived Lamina Propria Mononuclear Cells From Patients with Inflammatory Bowel Disease

We will use CITE-seq to characterize lamina propria mononuclear cells isolated
from intestinal mucosal biopsies obtained for research purposes during
colonoscopies performed for clinical purposes in patients with inflammatory
bowel disease (IBD). The proposed pilot project will expand the scientific
scope of the NIDDK IBD Genetics Consortium and complement the work that the
parent grant is funding.

Ancillary R01 liaison: Alexander Marson, PhD
