---
title: 'BREM-SC: a bayesian random effects mixture model for joint clustering single
  cell multi-omics data'
date: '2020-06-01'
publishDate: '2021-02-15T12:57:09.112495Z'
authors:
- Xinjun Wang
- Zhe Sun
- Yanfu Zhang
- Zhongli Xu
- Hongyi Xin
- Heng Huang
- Richard H. Duerr
- Kong Chen
- Ying Ding
- Wei Chen
publication_types:
- '2'
abstract: Droplet-based single cell transcriptome sequencing (scRNA-seq) technology,
  largely represented by the 10× Genomics Chromium system, is able to measure the
  gene expression from tens of thousands of single cells simultaneously. More recently,
  coupled with the cutting-edge Cellular Indexing of Transcriptomes and Epitopes by
  Sequencing (CITE-seq), the droplet-based system has allowed for immunophenotyping
  of single cells based on cell surface expression of specific proteins together with
  simultaneous transcriptome profiling in the same cell. Despite the rapid advances
  in technologies, novel statistical methods and computational tools for analyzing
  multi-modal CITE-Seq data are lacking. In this study, we developed BREM-SC, a novel
  Bayesian Random Effects Mixture model that jointly clusters paired single cell transcriptomic
  and proteomic data. Through simulation studies and analysis of public and in-house
  real data sets, we successfully demonstrated the validity and advantages of this
  method in fully utilizing both types of data to accurately identify cell clusters.
  In addition, as a probabilistic model-based approach, BREM-SC is able to quantify
  the clustering uncertainty for each single cell. This new method will greatly facilitate
  researchers to jointly study transcriptome and surface proteins at the single cell
  level to make new biological discoveries, particularly in the area of immunology.
featured: false
publication: '*Nucleic Acids Research*'
grcs: [pittsburgh]
doi: 10.1093/nar/gkaa314
---

