---
# Documentation: https://wowchemy.com/docs/managing-content/

title: A modular organization of the human intestinal mucosal microbiota and its association
  with inflammatory bowel disease
subtitle: ''
summary: ''
authors:
- Maomeng Tong
- Xiaoxiao Li
- Laura Wegener Parfrey
- Bennett Roth
- Andrew Ippoliti
- Bo Wei
- James Borneman
- Dermot P. B. McGovern
- Daniel N. Frank
- Ellen Li
- Steve Horvath
- Rob Knight
- Jonathan Braun
tags:
- '"Humans"'
- '"Inflammatory Bowel Diseases"'
- '"Intestinal Mucosa"'
- '"Cohort Studies"'
- '"Phenotype"'
- '"Microbiota"'
- '"Cluster Analysis"'
- '"Metagenome"'
- '"RNA"'
- '"Ribosomal"'
- '"16S"'
categories: []
date: '2013-01-01'
lastmod: 2021-11-01T08:41:31-04:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-11-01T12:41:31.490824Z'
publication_types:
- '2'
abstract: Abnormalities of the intestinal microbiota are implicated in the pathogenesis
  of Crohn's disease (CD) and ulcerative colitis (UC), two spectra of inflammatory
  bowel disease (IBD). However, the high complexity and low inter-individual overlap
  of intestinal microbial composition are formidable barriers to identifying microbial
  taxa representing this dysbiosis. These difficulties might be overcome by an ecologic
  analytic strategy to identify modules of interacting bacteria (rather than individual
  bacteria) as quantitative reproducible features of microbial composition in normal
  and IBD mucosa. We sequenced 16S ribosomal RNA genes from 179 endoscopic lavage
  samples from different intestinal regions in 64 subjects (32 controls, 16 CD and
  16 UC patients in clinical remission). CD and UC patients showed a reduction in
  phylogenetic diversity and shifts in microbial composition, comparable to previous
  studies using conventional mucosal biopsies. Analysis of weighted co-occurrence
  network revealed 5 microbial modules. These modules were unprecedented, as they
  were detectable in all individuals, and their composition and abundance was recapitulated
  in an independent, biopsy-based mucosal dataset 2 modules were associated with healthy,
  CD, or UC disease states. Imputed metagenome analysis indicated that these modules
  displayed distinct metabolic functionality, specifically the enrichment of oxidative
  response and glycan metabolism pathways relevant to host-pathogen interaction in
  the disease-associated modules. The highly preserved microbial modules accurately
  classified IBD status of individual patients during disease quiescence, suggesting
  that microbial dysbiosis in IBD may be an underlying disorder independent of disease
  activity. Microbial modules thus provide an integrative view of microbial ecology
  relevant to IBD.
publication: '*PloS One*'
doi: 10.1371/journal.pone.0080702
---
